lametro/models.py

```diff
@@ -409,31 +409,6 @@ def upcoming_committee_meetings(cls):
         return meetings
 
 
-    @property
-    def media(self):
```

Comment:
Replacing this with a template filter that sorts the media_urls set in the template, without hitting the database, again.

Reply:
Update! I am doing the ordering of media_urls in the original queryset.